pycbc/workflow/core.py

```diff
@@ -47,6 +47,15 @@
 from .configuration import WorkflowConfigParser, resolve_url
 from .pegasus_sites import make_catalog
 
+# FIXME: Are these names suitably descriptive?
```

Comment:
Note that there are two distinct "values" here, which are somewhat confused and confusing. See also the declarations on lines 73 - 76.

There's the global retention level "This workflow will keep jobs if they are level X or higher". For the global retention level (which these options are), lower numbers mean "keep more", so the level 0 choice here doesn't really make much sense, as level 0 should be "keep everything" at the global level. 

Then there's the local level (lines 73-76), which is "How important are the files produced by this Job?". In that case, higher means more likely to be stored. (From 4=always store me, to 0=never store me). I think a DO_NOT_KEEP level 0 option might belong at line 72, but not here.

Then I just need one more comment on a later line .....